Game.cs

```diff
@@ -2,28 +2,73 @@
 using Godot.Collections;
 using JetBrains.Annotations;
 
-[UsedImplicitly]
-public class Game : Node
+[UsedImplicitly] public class Game : Node
 {
     private AudioStreamPlayer musicPlayer;
     private Inventory inventory;
-    private Map map;
+    [CanBeNull] private Map map;
```

Comment:
map is now loaded dynamically (to be able to reload after game over) so it `CanBeNull`